src/schemas/ExportMnemonicDataSchema.ts

```
@@ -77,24 +77,24 @@ class ExportMnemonicDataSchema extends QRCodeDataSchema {
+      throw new Error('JSON argument cannot be empty.');
     }
 
     const jsonObj = JSON.parse(json);
     if (!jsonObj.type || jsonObj.type !== QRCodeType.ExportMnemonic) {
-      throw new Error("Invalid type field value for MnemonicQR.");
+      throw new Error('Invalid type field value for MnemonicQR.');
     }
 
-    if (!jsonObj.hasOwnProperty("data")) {
-      throw new Error("Missing mandatory property for encrypted payload.");
+    if (!jsonObj.hasOwnProperty('data')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)